assets/src/edit-story/components/panels/font.js

```diff
@@ -68,7 +72,26 @@ function FontPanel({ selectedElements, onSetProperties }) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [fontFamily, fontStyle, fontSize, fontWeight, getFontWeight]);
   const handleSubmit = (evt) => {
-    onSetProperties(state);
+    onSetProperties((properties) => {
+      const { width, height: oldHeight, rotationAngle, x, y } = properties;
+      const updatedState = removeUnsetValues(state);
+      const newProperties = { ...properties, ...updatedState };
+      const newHeight = dataPixels(calculateTextHeight(newProperties, width));
+      const [dx, dy] = calcRotatedResizeOffset(
+        rotationAngle,
+        0,
+        0,
+        0,
+        newHeight - oldHeight
+      );
+      return {
+        ...updatedState,
+        height: newHeight,
```

Comment:
Shouldn't this update width as well? E.g. if font size is increased then this would mean that the text would likely go over the borders of the element even though it would fit length-wise.

Reply:
Per UX for now, they decided to only update height. E.g. the width doesn't change, but the element grows/shrinks in height as needed to accommodate new wrapping. The width overflow is possible, of course. So I anticipate something might change in these requirements, but this is where they are at the moment.

Reply:
OK, sounds good. Can't seem to find this UX in Figma, could you share a link to where the new UX logic can be found?

I only found this which indicates as if both should change:
<img width="515" alt="Screenshot 2020-02-25 at 08 46 39" src="https://user-images.githubusercontent.com/3294597/75225677-6abcbe80-57ab-11ea-9077-526cda485792.png">

Reply:
Here ya go: https://docs.google.com/document/d/1_3-_v4zH6bW9hOplOO8wYxj8l4C0soMgEbD7XLiXrrg/edit#heading=h.v5b5x3xo1pxy